docs/admin/authorization/rbac.md

```diff
@@ -151,19 +162,80 @@ metadata:
   namespace: default
   name: pod-and-pod-logs-reader
 rules:
-  - apiGroups: [""]
-    resources: ["pods", "pods/log"]
-    verbs: ["get", "list"]
+- apiGroups: [""]
+  resources: ["pods", "pods/log"]
+  verbs: ["get", "list"]
+```
+
+#### Role examples
+
+Only the `rules` section is shown in the following examples.
+
+Allow reading the resource "pods" in the core API group:
+
+```yaml
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+```
+
+Allow reading/writing "deployments" in both the "extensions" and "apps" API groups:
+
+```yaml
+rules:
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ```
 
-## Referring to Subjects
+Allow reading "pods" and reading/writing "jobs":
 
-RoleBindings and ClusterRoleBindings bind "subjects" to "roles".
+```yaml
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["batch", "extensions"]
+  resources: ["jobs"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+```
+
+Allow reading a configmap named "my-config" (must be bound with a `RoleBinding` to limit to a single configmap in a single namespace):
+
+```yaml
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  resourceNames: ["my-config"]
+  verbs: ["get"]
+```
+
+Allow reading the resource "nodes" in the core group (because `Node` objects are cluster-scoped, this must be in a `ClusterRole` bound with a `ClusterRoleBinding` to be effective):
+
+```yaml
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch"]
+```
+
+Allow "GET" and "POST" requests to the non-resource endpoint `/healthz` and all subpaths (must be in a `ClusterRole` bound with a `ClusterRoleBinding` to be effective):
```

Comment:
- [ ] Here, I'd write the endpoint name as a quoted string rather set it in monospace text, since it's not a reserved word or token that we're documenting here.
> endpoint "/healthz" and